baseband/mark5b/tests/test_mark5b.py

```diff
@@ -682,6 +682,35 @@ def test_filestreamer(self, tmpdir):
             assert abs(fh.start_time - test_time) < 1.*u.ns
             assert fh.header0['frame_nr'] == 198
 
+    def test_sequentialfile(self, tmpdir):
+        """Tests writing and reading of sequential files.
+
+        These tests focus on reading and writing with lists.
+        """
+
+        # Use sample file as basis of a file sequence.
+        with mark5b.open(SAMPLE_FILE, 'rs', sample_rate=32*u.MHz,
+                         kday=56000, nchan=8, bps=2) as fh:
+            header = fh.header0.copy()
+            data = fh.read()
+            dtime = fh.stop_time - fh.start_time
+        data = np.r_[data, data, data, data, data]
```

Comment:
Very small detail, but I have a big cognitive dissonance with `np.r_` - it is just not at all obvious what it is supposed to mean.  Could you change to just using `concatenate`?

Reply:
Changed to `np.concatenate`.